decision-making/voting-on-proposals.md

```diff
@@ -0,0 +1,30 @@
+# Decision-making
+
+### Proposal submission
+Proposal is submitted by opening pull request to this repository.
+Initiator must present the idea on the architecture meeting.
```

Comment:
Does the initiator have to wait for the next meeting or is this requirement independent of the 2 week timeline given below?

Reply:
I would say, initiator must present the idea and give 2 weeks discussion time. Need to work on wording.

Reply:
Why does it need to be presented in a separate meeting? Or perhaps what types of changes need to go via this route and what kinf of changes can "just" go as PRs?